lib/messages/status.js

```diff
@@ -4,15 +4,24 @@ const {
 } = require('.');
 
 class Status extends Message {
-  constructor(status) {
+  constructor(checkOrStatus) {
     super({});
-    this.status = status;
+    /*  error, failure, pending, success, queued, in_progress,
+        completed, neutral, cancelled, timed_out, or action_required
+    */
+    this.overallStatus = checkOrStatus.state || checkOrStatus.conclusion || checkOrStatus.status;
+    this.context = checkOrStatus.context || checkOrStatus.app.name || checkOrStatus.name;
+    this.description = checkOrStatus.description ||
+      checkOrStatus.output.title ||
+      checkOrStatus.name;
+    this.avatar_url = checkOrStatus.avatar_url || checkOrStatus.app.owner.avatar_url;
+    this.target_url = checkOrStatus.target_url || checkOrStatus.html_url;
   }
 
   static getStatusColor(status) {
     if (status === 'success') {
       return constants.STATUS_SUCCESS;
-    } else if (status === 'pending' || status === 'in_progress' || status === 'queued') {
+    } else if (status === 'pending' || status === 'in_progress' || status === 'queued' || status === 'neutral') {
```

Comment:
Same than the previous comment. It could be a bit better to have these statuses in an array and then use `.includes()` but again, it's ok to leave it like it is by now.